src/amo/actions/reviews.js

```diff
@@ -42,9 +42,15 @@ export const DELETE_ADDON_REVIEW: 'DELETE_ADDON_REVIEW' = 'DELETE_ADDON_REVIEW';
 export const UNLOAD_ADDON_REVIEWS: 'UNLOAD_ADDON_REVIEWS' =
   'UNLOAD_ADDON_REVIEWS';
 
+export type SimplifiedAddonType = {|
```

Comment:
Could we call it `ReviewAddonType`? I think we have a similar type somewhere for the `addon` field in the response of a [search request](https://addons-server.readthedocs.io/en/latest/topics/api/addons.html#get--api-v4-addons-search-) because it too is missing some fields (but I can't find the type now).